examples/dates/DateTimeInfoExtractor.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
I think it would be good to include the the default mappings in here, as well as the more detailed information from the transformer docstring